KMeansUtil.py

```diff
@@ -0,0 +1,81 @@
+import math, utils, random
+
+class KMeansUtil:
+
+	#Constructor to initialize the value of K
+	def __init__(self):
+		self.k = int(input("Enter the number of clusters: "))
```

Comment:
Init functions are just used to initialize variables for an object. Pass it as a parameter to this function.